client/components/FAQ.scss

```diff
@@ -0,0 +1,118 @@
+.c_header{
+    margin-top:84px;
+    width: 607px;
+  height: 56px;
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.17;
+  letter-spacing: normal;
+  color: #01131f;
+  margin-left:64px;
+  margin-right:769px;
+}
+.c_rectangle_general_information{
+    margin-top:80px;
+    width: 862px;
+    height: 640px;
+    margin-left:64px;
+    padding-left:32px;
+    border-radius: 8px;
+    box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.16);
+    background-color: #fdfeff;
+    padding-top:40px;
+    margin-bottom:80px;
+}
+.c_gen_information{
+    text-align:left;
+    margin-bottom:24px;
+    width: 288px;
+    height: 40px;
+    font-family: Roboto;
+    font-size: 32px;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 1.25;
+    letter-spacing: normal;
+    color: var(--black);
+}
+ul{
+    padding:0px;
+    margin:0px;
+
+}
+li{
+    list-style-type: none;
+    margin-right:32px;
+}
+button{
+        width: 798px;
+        height: 80px;
+        background-color: #f6f6f7;
+        border-style:none;
+        text-align:left;
+        font-family: Roboto;
+        font-size: 24px;
+        font-weight: bold;
+        font-style: normal;
+        font-stretch: normal;
+        line-height: 1.33;
+        padding-left:24px;
+        margin-bottom:24px;
+
+}
+.c_rectangle_security{
+        width: 862px;
+        height: 536px;
+        border-radius: 8px;
+        box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.16);
+        background-color: #fdfeff;
+        margin-bottom:80px;
+        margin-left:64px;
+        padding-left:32px;
+        padding-top:40px;
+}
+.c_security {
+    width: 119px;
+    height: 40px;
+    margin-bottom:24px;
+    font-family: Roboto;
+    font-size: 32px;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 1.25;
+    letter-spacing: normal;
+    color: var(--black);
+  }
+.c_rectangle_value{ 
+        width: 862px;
+        height: 432px;
+        padding-top:40px;
+        border-radius: 8px;
+        box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.16);
```

Comment:
Look into [SASS variables](https://sass-lang.com/documentation/variables)--these should allow you to avoid repeating the declarations.

Reply:
Am I correct SASS variables are stored at the beginning of code and then recalled from memory? for example 24px is often reused, but storing it in a variable would do the same thing as explicitly defining it in the object. Either way, whether you store it in local memory, or have the computer render it on the fly, it is doing the same thing, and taking up more space by having a defined variable called every time you need to use it. One way the processor has to call the variable, the other it just renders it from the value... I don't understand why I would need to do the extra work because there shouldn't be any noticeable difference in the cost of processing power

Reply:
The advantage of variables isn't in processing power--this gets compiled down to CSS at build time anyway, and the variable space is negligible. The advantage is in code maintainability: for example, let's say we want to change the color of all the box shadows. With a variable, we'd just need to change the value of the variable, whereas with repeated code, we'd have to find all occurrences of the box-shadow declaration and update them manually (and we could miss some, or accidentally introduce a typo on one, etc.).